pkg/cmd/cli/cmd/newapp.go

```diff
@@ -70,7 +70,7 @@ application is created.`
 )
 
 // NewCmdNewApplication implements the OpenShift cli new-app command
-func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+func NewCmdNewApplication(fullName string, f *clientcmd.Factory, out, errOut io.Writer) *cobra.Command {
```

Comment:
Good catch. Seems like upstream could do the same?
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubectl/cmd/exec.go#L46